keymap/user/gkbd/fnrs.kbd

```diff
@@ -0,0 +1,31 @@
+(defcfg
+  ;; ** For Linux **
+  input  (device-file "/dev/input/by-id/usb-USB_KEYBOARD-event-kbd")
```

Comment:
Are you actually using these configuartions yourself?  These seem more like templates for this particular keyboard layout rather than actual user configuration files

Reply:
I have multiple keyboards and did not want to keep actual device ID in repo, hence generic placeholder.